dvc/logger.py

```diff
@@ -155,3 +156,65 @@ def debug(cls, msg, exc=None):
     @staticmethod
     def info(msg):
         Logger._with_progress(Logger.logger().info, msg)
+
+    @classmethod
+    def box(cls, msg, border_color=''):
+        lines = msg.split('\n')
+        max_line_length = max(len(cls._remove_colors(line)) for line in lines)
+
+        # Spaces between the borders and the content
+        padding_horizontal = 5
+        padding_vertical = 1
+
+        box_size_horizontal = (max_line_length + (padding_horizontal * 2))
+
+        chars = {
+            'top_left':     '┌',
```

Comment:
If I remember correctly windows cmdline has some issues with unicode. Need to try this out, just to make sure. If it indeed doesn't support it, then we can use some ASCII substitutes(e.g. in dvc/dagascii.py we draw boxes using +,|,-).

Reply:
I know that windows comandline is not commonly used by itself as a workspace for such projects, but it is for us as IE for web developers, so we need to make sure it works :)

Reply:
Nice, @efiop , I'll take that into account!

Reply:
According to this answer in stack overflow, it shouldn't be problem outputting those unicode, @efiop 
https://stackoverflow.com/a/47843552

Reply:
I ended using ASCII characters, @efiop